inst/templates/validation_packet.Rmd

```diff
@@ -61,7 +61,7 @@ all_sig <- vt_scrape_sig_table()
 **AUTHOR** My signature designates authorship of this document.
 
 ```{r validation-lead-sig-block}
-all_sig[tolower(all_sig$role) == "validation lead",] %>%
+all_sig[grepl("validation lead",all_sig$role, ignore.case = TRUE),] %>%
```

Comment:
I think in the future we could extend this further by pulling the 'validation lead' title from the config yaml.

Reply:
opened issue #200 for discussion